features/step_definitions/database.rb

```diff
@@ -1,11 +1,35 @@
+require 'fileutils'
 require 'aruba/cucumber'
 require 'sqlite3'
 
-Then('we can open a SQLite connection to {string}') do |database|
-    Aruba.configure do |config|
-        database["~"] = config.home_directory
-        db = SQLite3::Database.open database
-        r = db.execute("PRAGMA integrity_check")
-        expect(r[0]).to match(["ok"])
+Given('that I ran {string}') do |command|
+  system command
+end
+
+Given('a blank file named {string} exists') do |filename|
+  Aruba.configure do |config|
+    if filename.include? "~"
+      filename["~"] = config.home_directory
+    end
```

Comment:
The logic for expanding the `~` into the home directory is repeated a few times in this file. I think it makes sense to add a small method to do this.

```ruby
def expand_home_directory(filename, home_directory)
  if filename.include? "~"
    filename['~'] = home_directory
  end
  filename
end
```

And then this call site would change to:

```suggestion
    filename = expand_home_directory(filename, config.home_directory)
```